### ISSUE TITLE
Integrate Cancer Biomarker Database

### ISSUE THREAD

**Issue by ckongEbi** [opentargets/data_release#49]
The idea is to integrate data from https://cancergenomeinterpreter.org/biomarkers into the target profile page.

Tasks (to be expanded) :
- [ ] Look at the data_pipeline, gene merging step `--gen` , the plugin system
- [ ] Enrich/Map the initial dataset with Ensembl ID (biomarker) , Chembl ID (drug), Disease ID (Tumor type)

**Comment by elipapa** [comment]
Issue moved to [opentargets/data_pipeline #229](https://github.com/opentargets/data_pipeline/issues/229) via [**ZenHub**](https://www.zenhub.com/)

**Comment by MichaelaEBI** [comment]
I started an 'Initial Data Specifications' Document for the Cancer Biomarker Database: https://docs.google.com/document/d/1hRj66U89Z8F3P0F-ruYKp1ExB1p3pC9TVfL1027zy7g/edit#. I looked into the data in more detail and I think that before we can look at back end or front end requirements and potential problems, we need to answer the Why? and What? Questions - why do we want to include this data in the OT platform and how should we go about it, i.e., where and how does it fit?
